src/App.tsx

```diff
@@ -1,26 +1,13 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from 'react';
+import { Todo } from './Components/Todo/Todo';
+import './App.scss';
+
+export const App = () => {
+  const [backGround, setBackGround] = useState(true);
 
-function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className={backGround ? 'App light' : 'App dark'}>
```

Comment:
It'll be better to use consts for "Light", "Dark" for `backGround`
or if you want to have boolean variable, then smth like `isDarkBackground`